sycl/source/detail/queue_impl.hpp

```diff
@@ -377,20 +386,46 @@ class queue_impl {
   /// \return a native handle.
   pi_native_handle getNative() const;
 
+  bool kernelUsesAssert(const std::string &KernelName,
+                        OSModuleHandle Handle) const;
+
+  void asynchronouslyDeleteBuffer(buffer<AssertHappened, 1> *B);
```

Comment:
Minor. Could you please clarify why this function is needed?

Reply:
This is rudimentary change and is not needed.
Removed it.